src/integration_fleets.md

```diff
@@ -31,11 +34,53 @@ The basic workflow of developing a fleet adapter is the following:
     - You will also need to provide a callback that will be triggered when the adapter is finished registering the robot. This callback will provide you with a new `RobotUpdateHandle` for your robot. It is imperative to save this update handle so you can use it to update the robot's position over time.
 8. As new information arrives from the fleet manager API, use the collection of `RobotUpdateHandle` classes to keep the adapter up-to-date on the robots' positions.
 
+With the robots controlled and managed by their respective fleets, robots of different models with varying configurations (such as task capabilities, vehicle traits and communication interface) can coexist and operate simultaneously in the same environment under RMF.
```

Comment:
This statement seems a bit out of place. We can either move it to the `Intro` page under `What is RMF` or leave it out.

Reply:
Ok! Removed it as i think the intro explained the same idea